integration/simulation/validate_chain.go

```diff
@@ -378,11 +377,19 @@ func checkLogsReceived(t *testing.T, s *Simulation) {
 		sub.Unsubscribe()
 	}
 
+	for _, channel := range s.LogChannels {
+		checkReceivedHOCAndPOCLogs(t, channel)
```

Comment:
We now check that each account has received HOC and POC events, and not the overall network.